### DISCUSSION TITLE
element not found and 90% when updating wsl

### DISCUSSION THREAD

**Discussion by adharidwan** [microsoft/WSL#11986] [Q&A]
before i was using a wsl for ubuntu 20.04 then i uninstalled it then i activaed secure boot (to play valorant), now i want to use wsl for kali-linux but when i tried to wsl --update it showed
Installing: Windows Subsystem for Linux
Element not found.=========90,0%=====================      ]
is there anybody know how to fixt it?

**Comment by adolphlaoma** [comment]
quote from Microsoft community：

installed the latest wsl version directly from microsoft's github.


https://github.com/microsoft/WSL/releases

click on the one which contains "latest" and download the .msi which corresponds to your system (if you're using an intel/amd processor, it will be x64. case you have a snapdragon pc, use ARM)

this will manually install the latest version of wsl in your machine.

hope it helps
